src/GDAL.jl

```diff
@@ -30,7 +30,8 @@ module C
 end
 
 include("C/misc.jl")
-include("C/common.jl")
+include("types.jl")
+include("common.jl")
```

Comment:
I just added commit 34e16701c0c89a717a8c14bd73476e1e57ad4102.
This removes all `typealias ANONYMOUS_*` from `src/common.jl`.

Currently the main module is not dependent on the C submodule, and since quite a few of the constants are needed in normal use (like `GDAL.GDAL_OF_VECTOR`), I'm not sure whether we should put all this in the C submodule.

Perhaps I don't fully understand and you can clarify. Are you talking only about the typealiases in `src/common.jl`, or about the whole file. We can also do a better job rewriting `src/common.jl`. `src/types.jl` is basically already a manual rewrite of parts of common.